README.md

```diff
@@ -17,6 +17,9 @@
     <a href="https://github.com/devicons/devicon/stargazers">
         <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/devicons/devicon?color=%2360be86&label=github%20stars&style=for-the-badge">
     </a>
+    <a href="https://github.com/devicons/devicon/stargazers">
+        <img alt="Registered icons" src="https://img.shields.io/github/directory-file-count/devicons/devicon/icons?color=%2360be86&label=registered%20icons&style=for-the-badge">
+    </a>
```

Comment:
If we talk logos/Icons, here is the number of logos. I'll look if there's a way to count the icons, but I'm not sure, as we must computes what's declared inside the `devicon.json`

Reply:
I don't think we can have that option : https://shields.io/category/size

Reply:
These two seem like they can do the job:
- GitHub repo directory count (custom path):	GitHub repo directory count (custom path) badge	/github/directory-file-count/:user/:repo/:path?type=dir
- GitHub repo file count (custom path & file type):	GitHub repo file count (custom path & file type) badge	/github/directory-file-count/:user/:repo/:path?type=file

If not, making our own endpoint would also be a possibility, unless dynamic shields can work for us.
![Screenshot_20220715_122322](https://user-images.githubusercontent.com/43886029/179205149-dc0c534e-824f-49b6-aa7c-96c2b0f8773b.jpg)

https://shields.io/endpoint

Reply:
Going to look into that thanks

Reply:
> These two seem like they can do the job:
> 
> * GitHub repo directory count (custom path):	GitHub repo directory count (custom path) badge	/github/directory-file-count/:user/:repo/:path?type=dir
> * GitHub repo file count (custom path & file type):	GitHub repo file count (custom path & file type) badge	/github/directory-file-count/:user/:repo/:path?type=file
> 
> If not, making our own endpoint would also be a possibility, unless dynamic shields can work for us. ![Screenshot_20220715_122322](https://user-images.githubusercontent.com/43886029/179205149-dc0c534e-824f-49b6-aa7c-96c2b0f8773b.jpg)
> 
> https://shields.io/endpoint

I did try the two suggested URL. But what we are missing is the total count of every files inside every directory in `icons`. And those two count only in the root.

I looked into the dynamic endpoint, we could use it to count inside the devicon.json, in a future because of this issue : https://github.com/badges/shields/issues/6071.

Our own endpoint could be the last solution right now, but we need to do it cleverly to avoid maintenance issues.

Reply:
@kilian-paquier How is this looking, any progress? 
Should I mark the PR as stale, or do you want to work on it further?

Reply:
Hello, I didn't investigate since my last message on this thread. Maybe the JSON endpoint, but we may have to maintain the number manually ...

Reply:
That should be possible to do, but maybe we can just add it in the future instead. I think we have enough for this feature 💯

Reply:
Yes, at least for the README part

Reply:
Resolving this, since it seems we've done all of the changes we want so far. Feel free to unresolve if something's still missing